config/development.yaml

```diff
@@ -1,6 +1,7 @@
 persistence:
   defaultStore: cass-default
   visibilityStore: cass-visibility
+  advancedVisibilityStore: es-visibility
```

Comment:
I added comment to make it explicate that this is optional. 
But I didn't remove it from development because I think it is most convenient for dev and docker for now.

Reply:
Let's remove it from development config and move it to specific elastic search config.  This is an optional component at this point and we want to keep development config focussed on minimal components needed for functional Cadence setup.